examples/Slider.labels.tsx

```diff
@@ -3,19 +3,25 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { Slider } from '@itwin/itwinui-react';
+import { Slider, Text } from '@itwin/itwinui-react';
+import { useId } from '../packages/itwinui-react/src/core/utils/hooks/index.js';
 import {
   SvgSmileyHappyVery,
   SvgSmileySadVery,
 } from '@itwin/itwinui-icons-react';
 
 export default () => {
+  const labelId = useId();
+
   return (
     <div style={{ width: 'min(100%, 300px)' }}>
+      <Text id={labelId} as='label' variant='leading'>
```

Comment:
`<Label>` styling should work better than `<Text variant='leading'>`.

also not sure if we should use `as='label'` or `as='div'`.

Reply:
> `<Label>` styling should work better than `<Text variant='leading'>`.

https://github.com/iTwin/iTwinUI/pull/1474/commits/0eb23657b5e8e0d71dc948c55cc879621677db8e: Ohh, nice. Yes, `<Label>` styling looks great. It also now looks more similar to the label in LabeledSelect.

> also not sure if we should use `as='label'` or `as='div'`.

https://github.com/iTwin/iTwinUI/pull/1474/commits/03ac73f11df68437e7a7cdf8f20b1ee6248596e8: Since the slider is actually a `<div role="slider">` and not an `<input type="range">`, I changed the `as` of `<Label>` to be a `div` and not a `label`. I found [this example on the MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role#examples) to confirm this pattern of:

| Label   | Slider|
|----------|-------------|
| `<label for="sliderId">` |  `<input id="sliderId" type="range">`  |
| **`<div id="labelId">`** | **`<div role="slider">`** (our iTwinUI Slider) |